### PR TITLE
Adds One Shot RPG to Black Market

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
as title

## Why It's Good For The Game
Purchasable Anti-Armor for crews that might not otherwise have access to it.

## Changelog

:cl:
add: A cargo ship happened to lose an entire crate of Hammer Rocket Launchers while travelling through the system. We have reason to believe they're probably on sale now.
del: you can no longer purchase PML-9s on the black market.
/:cl:

### CODE REVIEW

**Comment by rye-rice** [comment]
can you rewmove the other black market rpg while youre at it

**Comment by Erikafox** [comment]
oh sure

File: code/modules/cargo/blackmarket/blackmarket_items/explosives.dm [hunk 1]
```diff
@@ -97,18 +97,6 @@
 	stock_max = 7
 	availability_prob = 50
 
-/datum/blackmarket_item/explosive/rpg
```
Comment (SomeguyManperson) [inline code/modules/cargo/blackmarket/blackmarket_items/explosives.dm:100]: Undocumented

Comment (Erikafox) [reply code/modules/cargo/blackmarket/blackmarket_items/explosives.dm:100]: Updated cl

**Pull request merged**